### DISCUSSION TITLE
Adding additional products to OrderDetail when specific products are selected

### DISCUSSION THREAD

**Discussion by mangix21** [serenity-is/Serenity#5554] [Help]
Hey guys,

I'm looking to add functionality to my ordering process where when I select say "Metal Part A" and add it to my order details, it automatically adds Products Y and Z along with it to the order, and maybe even accounts for the quantity as well.

I'm trying to make an application for Metal related ordering and a lot of the time certain products also come along with Vinyl and screws. It would be really efficient if instead of manually mathing out having 6x of Metal Part A, so I need to go back in and add Vinyl quantity 300 (50 per 1 metal part) and screws quantity 120 (20 per 1 metal part). The ordering process was Add 6 metal pieces, and when the page goes back to order detail list I have 6 of A, 300 vinyl, and 120 screws in my list.

Does anyone know if similar functionality is already built in and I'm just missing it, or an easy way to add?
I'm assuming I would need to add a bool or two to the products table like "Has vinyl" "has screws" and then go from there, but I'm not entirely certain.

---

edit: It looks like this may be the best starting place.
https://serenity.is/demo/AdvancedSamples/ProductPicker#
So I'll go from there. If anyone has a different link that might requires less tinkering feel free to drop it. Thanks.

**Comment by choss21** [comment]
Ando trabajando en una funcionalidad Muy similar para Pedidos

Con Serenity.Extra agregue Edicion inline y quedo muy bien

Saludos
